manim/_config/default.cfg

```diff
@@ -146,3 +147,14 @@ log_timestamps = True
 # Uncomment the following line to manually set the loglevel for ffmpeg. See
 # ffmpeg manpage for accepted values
 loglevel = ERROR
+
+
+# Streaming settings
+# I'm using rtp because I've tried it and it works
```

Comment:
can be removed before merge.

Reply:
Presuming there's no other protocol one may accept as default.
I'm wont to see myself in the wrong so I'll just take all your words in good faith from here

Reply:
```suggestion
# Protocol defaults to rtp.
```
Maybe do something like this? Point is, these are your personal comments, which imo are a bit confusing to end users.

Reply:
My friend was right. I don't write end user code

Reply:
I think I made this commit with the rest of the changes

Is this streaming working for you?

Reply:
Yep! Though I agree with behackl, the stream should start automatically in order to make it easier to use. After that ig it should be ok to merge. I did have a slight issue when experimenting with it, but I'll add that as a separate comment.

Reply:
Wait it should start automatically

I wrote the code for that. Although if you're talking about the window not popping up, either a quick `manim.wait()` should do the trick. Or am I speaking of the same thing?

Reply:
Oh okay, `manim.wait()` worked. Better document that ig.

The docs test seems to be a timeout, shouldn't be anything related to this PR though, so no need to worry.

Reply:
> `manim.wait()` worked

Wait nevermind I can add that popup myself. Should be the last commit*

*_what I said 30 commits ago_